gui/.env

```diff
@@ -14,7 +14,7 @@ VUE_APP_GUI_POLLING_INTERVAL=10000
 VUE_APP_WS_PORT=28100
 
 # set app info
-VUE_APP_BRANDNAME=CORTX
+VUE_APP_BRANDNAME=cortx
```

Comment:
Are we using this only for comparision purpose or showing it on UI too... If we are showing the same text on Header or somewhere it should be CORTX not cortx

Please confirm

Reply:
I guess its only for comparison purpose. I can see only one instance it used as for comparison. Angvish  please confirm

Reply:
Yes, pranay, this is just used for selecting the brand-name of the the .css file. Nothing else

Reply:
please take care from next time... Only the reviewer who have asked the query can mark it resolved... no one else... In exceptional scenario it may be me or Ajay can overrride it with comments